src/PrestaShopBundle/Controller/Admin/Sell/Catalog/SupplierController.php

```diff
@@ -488,11 +485,11 @@ private function getErrorMessages()
             ),
             UploadedImageConstraintException::class => [
                 UploadedImageConstraintException::EXCEEDED_SIZE => $this->trans(
-                    'Max file size allowed is "%s" bytes.', 'Admin.Notifications.Error', [
+                    'Maximum image size: %s.', 'Admin.Global', [
                     $iniConfig->getUploadMaxSizeInBytes(),
                 ]),
                 UploadedImageConstraintException::UNRECOGNIZED_FORMAT => $this->trans(
-                    'Image format not recognized, allowed formats are: .gif, .jpg, .png',
+                    sprintf('Image format not recognized, allowed format(s): .%s', 'gif, jpg, png'),
```

Comment:
Hi there! Actually, I'm still thinking about this wording... could the `%s` variable be blank? My point is: do you know if there is only one image format allowed here or if there are many?

Reply:
@matks how should we properly get available extensions? currently they are hard-coded into legacy `ImageManager`. One option would be to pass them to exception and get it later-on, for example in controller for translation in this case. It would definitely require separate pr, as it would touch all images validation though. wdyt?

Reply:
Otherwise, we can just keep _Image format not recognized, allowed formats are: .gif, .jpg, .png_ in _Admin.Notifications.Error_, I'm okay with that. But if we can precise it, it is awesome. ;-)

Reply:
@zuk3975 As the available extensions could be very different, your idea to pass them into exception is good 😄 👍 . The only requirement I add is : this must be an optional feature. Developer must be able to throw UploadedImageConstraintException without knowing the right extensions. He can provide them if he knows it, but it should not be mandatory.

Reply:
@matks, I leave it as it was then, image/file upload exceptions should be modified in separate PR

Reply:
> Otherwise, we can just keep _Image format not recognized, allowed formats are: .gif, .jpg, .png_ in _Admin.Notifications.Error_, I'm okay with that. But if we can precise it, it is awesome. ;-)

@LouiseBonnard waiting for wording approval if it's okay for current PR to leave as it was.